src/popup/pages/AccountDetailsTokens.vue

```diff
@@ -34,12 +34,12 @@ export default defineComponent({
 .token-list-wrapper {
   position: relative;
   padding-top: 4px;
+  margin-inline: calc(-1 * var(--screen-padding-x));
 }
 
 .search-bar-wrapper {
   background: var(--screen-bg-color);
-  margin-left: calc(-1 * var(--screen-padding-x));
-  margin-right: calc(-1 * var(--screen-padding-x));
+  margin-inline: calc(-1 * var(--screen-padding-x));
```

Comment:
Is this `.search-bar-wrapper` even usable? I mean it's in the scoped styles block and this class does not exists in the current scope. So I think it could be removed.